ui/imports/shared/popups/NicknamePopup.qml

```diff
@@ -61,7 +63,10 @@ StatusModal {
                 validationMode: StatusInput.ValidationMode.IgnoreInvalidInput
                 validators: [
                     StatusRegularExpressionValidator {
-                        regularExpression: /^[0-9A-Za-z_-]*$/
+                        validatorObj: RXValidator { regularExpression: /^[\w\d_-]*$/u }
```

Comment:
Shouldn't we make `RXValidator` a default `validatorObj` of `StatusRegularExpressionValidator` ?

Reply:
Haha, fell into that trap too :)

I had it originally exactly that way... but no can do, since you can't import DOtherSide from StatusQ, the importchecker chokes on it

Reply:
Ah..got it. I [had the same issue](https://github.com/status-im/StatusQ/pull/787/) with SortFilterProxyModel. We should link DoOtherSide with the StatusQ sandbox app and importchecker. It makes a lot of sense to use SortFilterProxyModel in StatusQ components, as well as DoOtherSide exposed utilities.

Btw. great to see importchecker working @micieslak :clap: